content/talks/algorithmic-art.yaml

```diff
@@ -0,0 +1,18 @@
+type: talk
+title: Art algorithmique
+date: 10/12/2015
+image: prompt_talks.png
+short: "Qu'est-ce que l'art algorithmique, et à quoi ça ressemble ?"
+replay_id:
+authors:
+  - terrienale
+description: "Qu'est-ce que l'art algorithmique, et à quoi ça ressemble ?"
```

Comment:
Could you find another description?
Something like "Je vous présente mon travail sur l'art algorithmique..."